source/includes/v2/_odsql.md

```diff
@@ -134,17 +134,66 @@ Supported geometry expressions are:
 
 A scalar function can be used in [select arithmetic expressions](#arithmetic-select-expression), [filter expressions](#filter-expression) or in the [group by clause](#group-by-clause).
 
-Function|Parameters|Description|Limitation
---------|-----------|----------|----------
-`length`|string literal or string field literal|Returns the number of characters|
-`now`|no parameter|Returns the current date|Only works on filter expressions
-`year`|date field literal|Returns the year of a date|
-`month`|date field literal|Returns the month of a date|
-`day`|date field literal|Returns the day of a date|
-`hour`|date field literal|Returns the hour of a date|
-`minute`|date field literal|Returns the minute of a date|
-`second`|date field literal|Returns the second of a date|
-`date_format`|date field literal|Returns a formatted date|
+Function|Parameters|Description
+--------|-----------|----------
+`length`|string literal or string field literal|Returns the number of characters
+`now`|optional named parameters (see below)|Returns the current date
```

Comment:
If you include a link to the desired section instead of "see below", it should better help readers find the relevant info.
```suggestion
`now`|optional named [parameters](#parameters-to-the-now-function)|Returns the current date
```